locale/en/download/package-manager.md

```diff
@@ -289,6 +290,28 @@ Or install a binary package (if available for your platform) using pkgin:
 pkgin -y install nodejs
 ```
 
+## nvm
+On Unix / OS X systems Node.js built from source can be installed by
+[nvm](https://github.com/creationix/nvm) by installing into the location that nvm expects:
+
+```bash
+$ make install DESTDIR='$(NVM_DIR)/versions/node/' PREFIX='$(FULLVERSION)-pre'
```

Comment:
> Could you run a separate bash script that sources nvm.sh, and output the version path, and run that from the Makefile?

Would be enough just to pass a dummy version to `nvm_version_path` and use the path, without the dummy version, and then send that path to make? 

Something along the lines of : 
```shell
$ make install DESTDIR=`nvm_version_path 999|sed s/999$//` PREFIX='$(FULLVERSION)-pre'
```